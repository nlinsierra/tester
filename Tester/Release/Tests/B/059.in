AH 7H 8H 4D 2S QD 6D 2C KS 2D KC TH 3D 3S QC AD JD JS 5S 7S
