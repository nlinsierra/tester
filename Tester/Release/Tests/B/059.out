32
3C 4C 5C 6C 7C 8C 9C TC JC AC 5D 7D 8D 9D TD KD 2H 3H 4H 5H 6H 9H JH QH KH 4S 6S 8S 9S TS QS AS
